.travis.yml

```diff
@@ -1,37 +1,39 @@
-sudo: required
```

Comment:
I think this is required for tarpaulin. Or at least it would be if we had any tests to run.

Reply:
sudo is [no-effect now](https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration).